src/modules/fw_autotune_attitude_control/fw_autotune_attitude_control_params.c

```diff
@@ -121,3 +121,83 @@ PARAM_DEFINE_INT32(FW_AT_AXES, 3);
  * @group Autotune
  */
 PARAM_DEFINE_INT32(FW_AT_MAN_AUX, 0);
+
+/**
+ * Frequency of the injected signal
+ *
+ * This parameter define frequency of
+ * signal measured in hertz
+ *
+ * @unit Hz
+ * @min 0.1
+ * @max 10.0
+ * @decimal 1
+ * @group Autotune
+ */
+PARAM_DEFINE_FLOAT(FW_AT_SYSID_FREQ, 1.0);
+
+
+/**
+ * Phase angle of the injected signal
+ *
+ * This parameter define phase angle of
+ * signal
+ *
+ * @unit rad
+ * @min 0.0
+ * @max 10.0
+ * @decimal 1
+ * @group Autotune
+ */
+PARAM_DEFINE_FLOAT(FW_AT_SYSIDPHASE, 0.0);
```

Comment:
Does the phase really matter for sys Id ?

Reply:
No I dont use it but maybe some academical works needs it